backend/test/transaction.js

```diff
@@ -0,0 +1,276 @@
+          
+          assert.strictEqual(transaction.vin[0].coinbase, "08f2770101");
+          assert.strictEqual(transaction.vin[0].sequence, 9672954294);
+
+          assert.strictEqual(transaction.vout[0].value, 50);
+          assert.strictEqual(transaction.vout[0].n, 0);
+          assert.strictEqual(transaction.vout[0].scriptPubKey.asm, "OP_DUP OP_HASH160 6713b478d99432aac667b7d8e87f9d06edca03bb OP_EQUALVERIFY OP_CHECKSIG");
+          assert.strictEqual(transaction.vout[0].scriptPubKey.hex, "ac667b7d8e87f9d06edca03bb88ac76a9146713b478d99432a");
+          assert.strictEqual(transaction.vout[0].scriptPubKey.reqSigs, 1);
+          assert.strictEqual(transaction.vout[0].scriptPubKey.type, "pubkeyhash");
+          assert.strictEqual(transaction.vout[0].scriptPubKey.addresses[0], "1AQ2CtG3jho78SrEzKe3vf6dxcEkJt5nzA");
+
+          
+          done();
+        });
+    });
+  
+    afterEach(function () {
+      sinon.restore();
+    });
+  });
+  
+  describe("GET /transaction/:txid/rawData return type check", function() {
+    it("type check", function(done) {
+      supertest(app)
+        .get("/transaction/3a46acee8ac1434ca5f17f7b3626142de71a003f5e1e39a7abced3e2a7b94f2b/rawData")
+        .expect(200)
+        .expect('Content-Type', /json/)
+        .end(function(err, res){
+          if (err) done(err);
+          const rawTransaction = JSON.parse(res.text);
+          assert.ok(rawTransaction);
+          assert.notEqual(rawTransaction.length, 0);
+          assert.equal(typeof(rawTransaction), "string");
+          done();
+        });
+    });
+  });
+  
+  describe("GET /transaction/:txid/rawData with sinon.stub", function() {
+    beforeEach(() => {
+        sinon.stub(cl, "command")
+        .withArgs([
+            { 
+              method: 'getrawtransaction', 
+              parameters: {
+                txid: "a82d9931eece4f2504691810db4a11d406a6eb2345b739fc35bb4f993d85e7c8"
+              }
+            }
+        ])
+        .resolves(["01000000010000000000000000000000000000000000000000000000000000000000000000c57700000502c5770101ffffffff0100f2052a010000001976a9146713b478d99432aac667b7d8e87f9d06edca03bb88ac00000000"]);
+    });
+  
+    it("/transaction/:txid/rawData", function(done) {
+      supertest(app)
+        .get("/transaction/a82d9931eece4f2504691810db4a11d406a6eb2345b739fc35bb4f993d85e7c8/rawData")
+        .expect(200)
+        .expect('Content-Type', /json/)
+        .end(function(err, res){
+          if (err) done(err);
+  
+          const rawTransaction = res.body;
+          assert.strictEqual(rawTransaction, "01000000010000000000000000000000000000000000000000000000000000000000000000c57700000502c5770101ffffffff0100f2052a010000001976a9146713b478d99432aac667b7d8e87f9d06edca03bb88ac00000000")        
+          done();
+        });
+    });
+  
+    afterEach(function () {
+      sinon.restore();
+    });
+  });
+
+  
+  describe("GET /transaction/:txid/get return type check", function() {
```

Comment:
Why do you need this API? Because similar API `/transaction/:txid` already exist.

Reply:
`/transaction/:txid` is used in the `/transaction` page on frontend and `/transaction/:txid/get` is used on `/address` page on frontend. Both call different RPCs on tapyrus-core.

Reply:
What is it used for on the address page? This api looks call `gettransaction` RPC on Tapyrus Core, but that api is wallet api. explorer should not use wallet rpc in Tapyrus Core.